.changeset/curly-wombats-wink.md

```diff
@@ -2,7 +2,6 @@
 "@ignored/hardhat-vnext-errors": patch
 "@ignored/hardhat-vnext-utils": patch
 "@ignored/hardhat-vnext": patch
-"@ignored/hardhat-vnext-build-system": major
```

Comment:
We no longer have this package, so I am removing it from changeset config and history.